### PR TITLE
Super admin

### PR DESCRIPTION
**Pull request by SaiAshish1729**

### CODE REVIEW

File: src/Routes/index.js [hunk 1]
```diff
@@ -8,6 +14,7 @@ const user = require("./v1/user.route");
 const restaurant = require("./v1/restaurant.route");
 const admin = require("./v1/admin.route");
 const integrated = require("./v1/integrated.route");
+>>>>>>> ac7d63706f79cf92b18f19238cacfaebcf151dda
```
Comment (arka1997) [inline src/Routes/index.js:17]: This should be removed

File: src/Routes/index.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
```
Comment (arka1997) [inline src/Routes/index.js:1]: And this also

**Comment by arka1997** [comment]
@SaiAshish1729 Please resolve the conflicts, will approve and merge it to master

**Review by arka1997 — APPROVED** [review]
Looks good to me

File: prisma/migrations/20241004162618_update/migration.sql
Comment (SAKET687) [inline prisma/migrations/20241004162618_update/migration.sql:1]: These changes are already present in the `migration.sql` file within the `20240930113227_follower_table_created` folder, so it seems unnecessary to include this migration file.

Comment (SaiAshish1729) [reply prisma/migrations/20241004162618_update/migration.sql:1]: There is an update in credential

File: src/Routes/admin.js
Comment (SAKET687) [inline src/Routes/admin.js:1]: This `admin.js` file should be placed within the `v1` folder inside the `Routes` folder. Please check the master branch for reference.

File: src/seeder/admin-seeder.js
Comment (SAKET687) [inline src/seeder/admin-seeder.js:1]: This file should be named `admin.seeder.js` instead of `admin-seeder.js`.

**Pull request merged**